### ISSUE TITLE
[BUG] Pig Spawner placed when trying to place REAL type block with real id 63 and 127

### ISSUE THREAD

**Issue by Golshr** [LoneDev6/ItemsAdder#186]
**Describe the bug**
A REAL type block doesn't render properly and is replaced by pig spawner

**To Reproduce**
REAL type blocks with ids 63 and 127 in items_ids_cache.yml are replaced by pigspawner when placed. Exception trace below.

**Probably useless informations**
I have a lot of blocks in a yml file that has TILE and REAL blocks in a random order. Of course, I tried to change blocks order in the yml file and erased cache, but again, the block with id 63 and 127 glitched.

**Expected behavior**
The block should render properly

**Info (please complete the following information):**
 - Server version: Spigot 1.15.2
 - Client version: Vanilla 1.15.2 + OptiFine 1.15.2 HD U G1 pre13
 - Plugin version: 2.0.9

**Additional context**
```
[22:01:46] [Server thread/ERROR]: Could not pass event BlockPlaceEvent to ItemsAdder v2.0.9

org.bukkit.event.EventException: null

	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:319) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at org.bukkit.craftbukkit.v1_15_R1.event.CraftEventFactory.callBlockPlaceEvent(CraftEventFactory.java:331) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.ItemStack.placeItem(ItemStack.java:207) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:481) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1251) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:27) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:1) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:19) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(SourceFile:144) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(SourceFile:118) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:918) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:911) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeAll(SourceFile:103) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:894) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:828) [spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_202]

Caused by: java.lang.ClassCastException: org.bukkit.craftbukkit.v1_15_R1.block.data.CraftBlockData cannot be cast to org.bukkit.block.data.MultipleFacing

	at dev.lone.itemsadder.d.c.d.a(SourceFile:92) ~[?:?]

	at dev.lone.itemsadder.d.d.a.b.a.d.f(SourceFile:82) ~[?:?]

	at sun.reflect.GeneratedMethodAccessor109.invoke(Unknown Source) ~[?:?]

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_202]

	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_202]

	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:315) ~[spigot-1.15.2.jar:git-Spigot-a03b1fd-5838285]

	... 20 more
```

**Comment by LoneDev6** [comment]
Thanks for the report, I will fix that ASAP in 2.0.15 update (not 2.0.14)

**Comment by LoneDev6** [comment]
Thanks that should be fixed in 2.0.15, I will release it today, reopen this issue if it still happens.
Please before update delete the file: real_blocks_ids_cache.yml

Let me know :D
